airflow/plugins/operators/airtable_to_gcs.py

```diff
@@ -144,15 +144,13 @@ def __init__(
             air_base_name=air_base_name,
             air_table_name=air_table_name,
         )
-        if api_key:
-            self.api_key = api_key
-        else:
-            self.api_key = get_secret_by_name("CALITP_AIRTABLE_API_KEY")
+        self.api_key = api_key
 
         super().__init__(**kwargs)
 
     def execute(self, **kwargs):
-        self.extract.fetch_from_airtable(self.api_key)
+        api_key = self.api_key or get_secret_by_name("CALITP_AIRTABLE_API_KEY")
```

Comment:
I've not seen this syntax before 👀

Reply:
yeah it's technically not best practice but I like it (usually people recommend ternary instead)